<launch>

<param name="/phidgets_daq/update_rate_ms" value="4" type="int"/>
<param name="/phidgets_daq/ratiometric" value="False" type="bool"/>

<param name="/phidgets_daq/interpreter_path" value="/home/imager/catkin/src/phidgets_daq/demo/phidgets_daq_interpreter_functions.py" />

<node name="data_interpreter" pkg="phidgets_daq" type="daq_interpreter.py" output="screen"/>
<node name="data_acquisition" pkg="phidgets_daq" type="daq.py" output="screen" />


</launch>
